lib/matplotlib/backends/backend_wx.py

```diff
@@ -761,29 +763,6 @@ def _get_imagesave_wildcards(self):
         wildcards = '|'.join(wildcards)
         return wildcards, extensions, filter_index
 
-    def gui_repaint(self, drawDC=None, origin='WX'):
```

Comment:
Need to add an API change note (it's mostly an implementation detail so not going to ask a deprecation period, which probably wouldn't make sense anyways).